source/framework/external/tinyxml/tinyxmlparser.cpp

```diff
@@ -41,11 +41,7 @@ distribution.
 // is less flexible than it appears. Changing the entries
 // or order will break putstring.
 TiXmlBase::Entity TiXmlBase::entity[TiXmlBase::NUM_ENTITY] = {
-    {"&amp;", 5, '&'},
-    {"&lt;", 4, '<'},
-    {"&gt;", 4, '>'},
-    {"&quot;", 6, '\"'},
-    {"&apos;", 6, '\''}};
+    {"&amp;", 5, '&'}, {"&lt;", 4, '<'}, {"&gt;", 4, '>'}, {"&quot;", 6, '\"'}, {"&apos;", 6, '\''}};
```

Comment:
I think we shouldn´t perform clang formatting in external classes.

Reply:
Removed here 6a3055627424c8cf01eca47ca3144f7a50425650

Reply:
why?

Reply:
This is external but I believe modified by us, by @nkx